x-pack/plugins/observability/server/lib/annotations/bootstrap_annotations.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { CoreSetup, PluginInitializerContext, KibanaRequest } from 'kibana/server';
+import { PromiseReturnType } from '../../../../apm/typings/common';
```

Comment:
Super minor nit: I think it’d be nice to keep imports from other plugins to a minimum, or from a location that denotes sharing. Could this type be added to the Observability plugin directly? Or exported from the `apm` plugin in one of the locations that denotes it’s part of the shared / public contract?

Reply:
Moving `PromiseReturnType` to the observability plugin sounds like a good idea 👍